### PR TITLE
[wix-ui-tpa] FloatingDropdownBase - change aria-haspopup prop value

### PR DESCRIPTION
**Pull request by aarnoldaas**
We got request from @NeilWix to improve dropdown accessibility

### CODE REVIEW

**Comment by sivanharel** [comment]
Thanks!
Also here the builds are failed since your commit is not signed.

In order to contribute to the library, you need to first sign your commits. If you haven’t set-up your GitHub account for signed commits, please follow this guide: https://github.com/wix/wix-style-react/blob/master/packages/wix-style-react/docs/contribution/CREATE_PR.md#sign-commits

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 package-lock.json
 /.idea/
+wix-style-react.iml
```
Comment (sivanharel) [inline .gitignore:4]: I guess this was added by mistake. So please remove it:)

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_7046/

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_7046

**Review by sivanharel — APPROVED** [review]
Great thanks!
I will merge it after when the tests will pass.

**Pull request merged**